backend/src/test/java/com/github/mariakohnen/packbag/backend/controller/PackingListControllerTest.java

```diff
@@ -144,6 +145,33 @@ void postNewPackingList_whenNameIsNotGiven_shouldReturnException() {
                 .expectStatus().isEqualTo(400);
     }
 
+    @Test
+    void postNewPackingList_whenColorIsNotGiven_shouldSetColorAsDefault() {
+        //GIVEN
+        NewPackingListDto newPackingListDto = NewPackingListDto.builder()
+                .destination("Kyoto")
+                .build();
+        //WHEN
+        PackingList actual = webTestClient.post()
+                .uri("/api/packinglists")
+                .bodyValue(newPackingListDto)
+                .exchange()
+                .expectStatus().is2xxSuccessful()
+                .expectBody(PackingList.class)
+                .returnResult()
+                .getResponseBody();
+        //THEN
+        assertNotNull(actual);
+        assertNotNull(actual.getId());
+        PackingList expected = PackingList.builder()
+                .id(actual.getId())
+                .destination("Kyoto")
+                .color("#5f8bc0")
+                .build();
+        assertEquals(24, actual.getId().length());
+        assertEquals(expected, actual);
+    }
+
```

Comment:
TL;DR Good.

Longer: I asked myself for some time, if this is not tested in the other tests that were changed here slightly. But, no Stefan, the explicit test here would faild and show why the others also fail.